library/src/main/scala/scala/scalajs/runtime/RuntimeLong.scala

```diff
@@ -852,7 +846,10 @@ object RuntimeLong {
       hiReturn = remHi
       remLo
     } else {
-      js.Tuple4(quotLo, quotHi, remLo, remHi)
+      // AskToString
+      val quot = asUnsignedSafeDouble(quotLo, quotHi)
+      val remStr = remLo.toString
+      quot.toString + "000000000".jsSubstring(remStr.length) + remStr
```

Comment:
Can you please add a comment why `remHi` can be disregarded here? There currently is a non-explicit precondition that `AskToString` may only be used if `b` is large enough.